### DISCUSSION TITLE
Batched Faster Whisper captures sentence "Terima kasih kerana menonton!" though it is not in the audio file.

### DISCUSSION THREAD

**Discussion by toanhn** [SYSTRAN/faster-whisper#1048] [Q&A]
I have an audio file in Malay, Batched Faster Whisper captures the sentence "Terima kasih kerana menonton!" (means "Thanks for watching!") though it is not in the audio file. How can I resolve this issue?

**Comment by techspecshq** [comment]
> I have an audio file in Malay, Batched Faster Whisper captures the sentence "Terima kasih kerana menonton!" (means "Thanks for watching!") though it is not in the audio file. How can I resolve this issue?

Pls upload your audio file
